metrics.yaml

```diff
@@ -181,22 +181,16 @@ user:
     expires: never
 
   income:
```

Comment:
Since you're changing the type, you should rename this: maybe `raw_income` or `exact_income`?